.github/workflows/kubernetes_test.yaml

```diff
@@ -161,6 +176,8 @@ jobs:
           path: |
             ./tests_e2e/cypress/screenshots/
             ./tests_e2e/cypress/videos/
+            ./tests_e2e/playwright/videos/
```

Comment:
Thanks for the heads-up, I didn't know that. Here is the relevant part of the gitignore documentation:

> - If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular `.gitignore` file itself. Otherwise the pattern may also match at any level below the `.gitignore` level.
> - If there is a separator at the end of the pattern then the pattern will only match directories, otherwise the pattern can match both files and directories.
> - For example, a pattern `doc/frotz/` matches `doc/frotz` directory, but not `a/doc/frotz` directory; however `frotz/` matches `frotz` and `a/frotz` that is a directory (all paths are relative from the `.gitignore` file).

Since we currently have

```gitignore
# tests
videos/
```

we are indeed ignoring `videos` folders everywhere.